lib/src/design_system/widgets/atoms/avatar/mds_avatar.dart

```diff
@@ -0,0 +1,151 @@
+              bottom: -spacing1,
+              child: _avatarItem(avatarList![1], context, true),
+            ),
+            Positioned(
+              left: spacing10 + spacing10,
+              bottom: -spacing1,
+              child: _avatarItem(
+                  MdsAvatarType(avatarText: '+' + (avatarList!.length - 2).toString(), avatarType: AvatarType.stone),
+                  context,
+                  true),
+            ),
+          ],
+        ),
+      );
+    }
+    return Container();
+  }
+
+  /// single item of avatar
+  Widget _avatarItem(MdsAvatarType? mdsAvatarType, BuildContext context, bool haveBorder) {
+    return Container(
+      decoration: BoxDecoration(
+        color: mdsAvatarType?.avatarText != null && mdsAvatarType!.avatarText!.trim().isNotEmpty
+            ? _getAvatarTheme(mdsAvatarType.avatarType).keys.first
+            : secondary,
+        shape: BoxShape.circle,
+        border: Border.all(
+          width: spacing1,
+          color: haveBorder ? ColorToken.white : ColorToken.transparent,
+        ),
+      ),
+      child: Padding(
+        padding: p2 + p0_5 + (haveBorder ? p1 : p0),
+        child: mdsAvatarType?.avatarText != null && mdsAvatarType!.avatarText!.trim().isNotEmpty
+            ? MDSBody(
+                mdsAvatarType.avatarText!.trim(),
+                color: _getAvatarTheme(mdsAvatarType.avatarType).values.first,
+              ).copyWith(
+                context,
+                fontSize: fontSize17,
+                fontWeight: FontWeight.w600,
+              )
+            : Icon(
+                Icons.person_outline,
+                size: spacing6,
+              ),
+      ),
+    );
+  }
+
+  /// getting text color and background color of avatar based on
+  /// the type of avatar
+  Map<Color, Color> _getAvatarTheme(AvatarType? avatarType) {
+    Color avatarBackgroundColor = primary;
+    Color avatarTextColor = ColorToken.white;
+    if (avatarType == AvatarType.haldi) {
+      avatarBackgroundColor = warning;
+      avatarTextColor = warningDarker;
+    } else if (avatarType == AvatarType.stone) {
+      avatarBackgroundColor = secondary;
+      avatarTextColor = ColorToken.black;
+    } else if (avatarType == AvatarType.jamun) {
+      avatarBackgroundColor = accent2;
+    } else if (avatarType == AvatarType.tawak) {
+      avatarBackgroundColor = accent1;
+      avatarTextColor = accent1Darker;
+    } else if (avatarType == AvatarType.nimbu) {
+      avatarBackgroundColor = accent4;
+      avatarTextColor = accent4Darker;
+    } else if (avatarType == AvatarType.neem) {
+      avatarBackgroundColor = success;
+      avatarTextColor = successDarker;
+    } else if (avatarType == AvatarType.mirch) {
+      avatarBackgroundColor = alert;
+    } else if (avatarType == AvatarType.jal) {
+      avatarBackgroundColor = primary;
+    }
+    Map<Color, Color> _avatarTheme = {avatarBackgroundColor: avatarTextColor};
+    return _avatarTheme;
+  }
+}
+
+class MdsAvatarType {
```

Comment:
Please keep the convention to MDS.

Use MDSAvatarType instead of MdsAvatarType